exonum_launcher/launcher.py

```diff
@@ -163,6 +168,30 @@ def wait_for_start(self) -> None:
 
         self.launch_state.complete_config(self.config, result)
 
+    def unload_all(self) -> None:
+        """Unload all artifacts marked as unloaded."""
+        for artifact in self.config.artifacts.values():
+            if artifact.action != "unload":
+                continue
+
+            unload_request = self._supervisor.create_unload_request(artifact, self.config.actual_from)
```

Comment:
Why do we have to create N transactions? As far as I can see, the unload request is a part of `ConfigPropose` message, which can have multiple `ConfigChange`s.

Reply:
Yeah. Good catch :)